datadog/api/metrics.py

```diff
@@ -14,13 +14,16 @@ class Metric(SearchableAPIResource, SendableAPIResource):
     _METRIC_QUERY_ENDPOINT = '/query'
     _METRIC_SUBMIT_ENDPOINT = '/series'
 
+    _SUPPORTED_DATA_TYPES = (int, float, long)
+
     @classmethod
     def _process_points(cls, points):
         now = time.time()
-        if isinstance(points, (float, int)):
+        if isinstance(points, cls._SUPPORTED_DATA_TYPES):
```

Comment:
What about `Decimal`? Should we perhaps support all types that can be converted to `float` or `long`?